src/_pytest/_code/code.py

```diff
@@ -255,10 +256,10 @@ def __str__(self):
             line = "???"
         return "  File %r:%d in %s\n  %s\n" % (fn, self.lineno + 1, name, line)
 
-    def name(self):
+    def _name_get(self):
         return self.frame.code.raw.co_name
 
-    name = property(name, None, None, "co_name of underlaying code")
```

Comment:
this looks like python2.4 compatibility code -- can probably just use `@property` syntax

Reply:
this one too :)